src/storage/file_manager.c

```diff
@@ -8546,7 +8551,11 @@ file_temp_preserve (THREAD_ENTRY * thread_p, const VFID * vfid)
   entry = file_tempcache_pop_tran_file (thread_p, vfid);
   if (entry == NULL)
     {
-      assert_release (false);
```

Comment:
I think this assertion says we have to use `preserve` interface to pop out temp files from the transaction list. Do you happen to make a new interface to handle temp files without preserving? Can I know where to pop a temp file from a transaction list and clean up the file?

Reply:
I fixed it to revert it.